.github/workflows/lint.yml

```diff
@@ -7,6 +7,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
+          skip-pkg-cache: true
```

Comment:
Unfortunately, this is needed to avoid running into https://github.com/golangci/golangci-lint-action/issues/135.